src/map/utils/mobutils.cpp

```diff
@@ -513,10 +517,18 @@ namespace mobutils
             }
         }
 
-        PMob->addModifier(Mod::DEF, GetBase(PMob, PMob->defRank));
-        PMob->addModifier(Mod::EVA, GetEvasion(PMob));
-        PMob->addModifier(Mod::ATT, GetBase(PMob, PMob->attRank));
-        PMob->addModifier(Mod::ACC, GetBase(PMob, PMob->accRank));
+        PMob->addModifier(Mod::DEF, GetDefense(PMob, PMob->defRank));
+        PMob->addModifier(Mod::EVA, GetBase(PMob, PMob->evaRank));  // Base Evasion for all mobs
+        PMob->addModifier(Mod::ATT, GetBase(PMob, PMob->attRank));  // Base Attack for all mobs is Rank A+ but pull from DB for specific cases
+        PMob->addModifier(Mod::ACC, GetBase(PMob, PMob->accRank));  // Base Accuracy for all mobs is Rank A+ but pull from DB for specific cases
+        PMob->addModifier(Mod::RATT, GetBase(PMob, PMob->attRank)); // Base Ranged Attack for all mobs is Rank A+ but pull from DB for specific cases
+        PMob->addModifier(Mod::RACC, GetBase(PMob, PMob->accRank)); // Base Ranged Accuracy for all mobs is Rank A+ but pull from DB for specific cases
+
+        // Only mobs in dynamis can parry
+        if (PMob->isInDynamis())
```

Comment:
I know this PR isn't finshed, but if I can give my opinion, I would feel more confortable if mobs had a mobMod for "canParry".
Or even a database column if that aint posible.

My logic behind this is, for instance, easier to customize for server operators. Also, I'm sure retail mobs can parry in places other than dynamis.
For instance, maybe some helper NPCs, which are mobs at the end of the day. Or the Generals in Besieged

Reply:
Which one would you prefer? Either or would work. I tend to stay away from adding more columns to sql but that's just a personal preference.

Reply:
I would go with a modifier, personaly.

Reply:
I'll second the mobmod for this case. that follows the pattern we recently set for shield blocking too.

Reply:
I added the mod in. I'll update the rest of the pool mods if this looks good. Let me know

Reply:
Looks good to me